other/apidsl/tox.in.h

```diff
@@ -1543,6 +1546,47 @@ namespace file {
      * has no avatar.
      */
     AVATAR,
+
+    /**
+     * File type for a "small" image in png format. Sticker are inline images similar in intent to emoji, but should be
+     * their own 'message' and shouldn't share a line with existing or later text. In contrast with standard inline
+     * images, stickers should be made available by the client, static, and multi use. Allowing other clients to save
+     * bandwidth by storing stickers for later use.
+     *
+     * This image should have a file_id generated by `tox_hash` to allow clients to save this image file for reuse, and
+     * later reference. Because stickers are expected to be saved, receiving clients should send TOX_FILE_CONTROL_CANCEL
+     * when they already have the sticker indicated by file_id; and because of this sending clients can't make any
+     * assumptions when these file transfers are canceled with no data transferred.
+     */
+    STICKER_PNG,
+
+    /**
+     * File type for a "small" image in SVG format. Otherwise same as TOX_FILE_KIND_STICKER_PNG.
+     */
+    STICKER_SVG,
+
+    /**
+     * File type for an image that the sending client elects to display 'inline' with other messages. It is up to the
+     * receiving client to decide how to handle this file kind, but at the minimum the receiver should include some
+     * reminder/notice that this is an inline image, and not just a normal file transfer of an image.
+     */
+    INLINE_IMAGE_PNG,
+
+    /**
+     * Short audio file, expected to be a voicemail message from a contact. Clients should offer a way to play the audio
+     * clip in client, but at the very least need to treat this like how they would otherwise treat a standard textual
+     * message.
+     *
+     * The format for this file type must be in raw opus format: https://www.opus-codec.org/docs/
+     *
+     * Senders should expect that files for this filetype will be renamed by the receiving client, allowing the sender
+     * to use the filename as the bitrate selected for the audio.
+     *
+     * If clients wish to stream the voicemail message live to try to act like traditional tape recorded voicemail
+     * systems for call screening, they may set file_size to 0 for a live stream. Receiving clients should enforce a max
+     * length to avoid abuse. 10 seconds of voice audio at 32kb/s in roughly 1MB on disk.
+     */
+    VOICEMAIL_OPUS,
```

Comment:
Why does it need to be in opus format? Can't this be detected from the file data?